libcxx/docs/Status/Cxx2cPapers.csv

```diff
@@ -76,7 +76,7 @@
 "`P2422R1 <https://wg21.link/P2422R1>`__","Remove ``nodiscard`` annotations from the standard library specification","2024-06 (St. Louis)","|Complete|","19","``nodiscard`` attributes were kept as a conforming extension"
 "`P2300R10 <https://wg21.link/P2300R10>`__","``std::execution``","2024-06 (St. Louis)","","",""
 "","","","","",""
-"`P3136R1 <https://wg21.link/P3136R1>`__","Retiring niebloids","2024-11 (Wrocław)","","",""
+"`P3136R1 <https://wg21.link/P3136R1>`__","Retiring niebloids","2024-11 (Wrocław)","|Nothing To Do|","","libc++ has been implementing niebloids as customization point objects since they are initially added."
```

Comment:
This should still not be "norhing to do". That means there is no library-affecting change in the standard, not that we've already implemented it.

Reply:
Hmm, do we want to say Complete without version? CC @mordante.

Reply:
OOPS. I was probably wrong.

https://reviews.llvm.org/D116570 (63a991d0358970d76700d084f05eb95cd29234c0) seems to be the commit that implemented P3136R1 in LLVM 14.

Reply:
I'd just put the same number as when we've completed the implementation of the algorithms (assuming that was after we've switched to allowing everything as CPOs).